region/norfair/east.json

```diff
@@ -3585,6 +3643,26 @@
                       ]},
                     {"heatFrames": 200}
                   ]
+                },
+                {
+                  "name": "IBJ",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canIBJ",
+                    {"heatFrames": 800}
+                  ],
+                  "note": "Kill the enemies, then IBJ."
+                },
+                {
+                  "name": "Frozen Enemies",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canUseFrozenEnemies",
+                    {"heatFrames": 1200}
+                  ],
+                  "note": "Wait for the Alcoon to walk off the edge. Use it and the Multiviola as platforms."
```

Comment:
For this dboost either have the alcoon walkinto you at the height of your jump, or get iframes first and jump through it.
Harder to do than the others.